snapcraft/project/_project_options.py

```diff
@@ -17,122 +17,24 @@
 import logging
 import multiprocessing
 import os
-import platform
-import sys
 from contextlib import suppress
-from typing import List, Set  # noqa: F401
+from typing import List, Set, Union  # noqa: F401
 
 from snapcraft.internal import common, errors
 from snapcraft.internal.deprecations import handle_deprecation_notice
 from snapcraft.internal.os_release import OsRelease
 
 
-logger = logging.getLogger(__name__)
+from ._arch_mapping import (
+    _ARCH_TRANSLATIONS,
+    _HOST_CODENAME_FOR_BASE,
+    _HOST_COMPATIBILITY,
+    _get_platform_architecture,
+    _find_machine
```

Comment:
What is the benefit of extracting these if they're aren't encapsulated at all?

Reply:
I'm moving them back for now, as discussed offline, and we tackle do this properly in another PR.